web/styles/compose.css

```diff
@@ -488,6 +488,75 @@
     margin: 0 1px;
 }
 
+.compose-scrolling-buttons-container {
+    display: grid;
+    grid-template-columns:
+        [scroller-backward-start buttons-start] 48px [scroller-backward-end] minmax(
+            0,
+            1fr
+        )
+        [scroller-forward-start] 48px [scroller-forward-end buttons-end];
```

Comment:
Should we be writing 3em rather than 48px by default these days?

Reply:
Possibly. My thought is that this is probably worth a design discussion. 48px is a pretty generous target area, so what we ought to discuss is whether someone with a larger font-size setting but narrower viewport would benefit in any way from that area growing and thus obscuring more of the formatting buttons.

Reply:
I see. Yeah I can see the argument for a fixed width as a minimum click target size; we should just have a comment explaining the reasoning if that's what we are doing.

Reply:
Left comments about that value both in the CSS and JS.